pkg/kubelet/server/server.go

```diff
@@ -285,6 +285,7 @@ const pprofBasePath = "/debug/pprof/"
 
 // InstallDeguggingHandlers registers the HTTP request patterns that serve logs or run commands/containers
 func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
+	glog.Infof("Adding Debug Handlers")
```

Comment:
Thanks! It will be better to clarify what we are adding handlers to since kubelet is a complicated  component (e.g. server, manager, shims etc).